### ISSUE TITLE
MSA viz

### ISSUE THREAD

**Issue by vincerubinetti** [JRaviLab/molevolvr2.0#8]
![Image](https://github.com/user-attachments/assets/49fc4bc2-bc63-47b4-b11e-cd726c3d327c)

Use [Nightingale](https://ebi-webcomponents.github.io/nightingale/?path=/story/components-tracks-interpro-track-readme--page) to start.

Other possibilities for libraries if we run into issues with Nightingale:
https://cmdcolin.github.io/awesome-genome-visualization/?latest=true

See the following comment for guidelines on implementation:
https://github.com/JRaviLab/molevolvr2.0/pull/32#issuecomment-2408741783

**Comment by triksi254** [comment]
Hello @vincerubinetti could I be assigned this task? Please also share a little about the issue

**Comment by triksi254** [comment]
@vincerubinetti @jananiravi can I still work on it without the outreachy label on it. please advice

**Comment by jananiravi** [comment]
Yes, let us know if you have any Qs!

**Comment by triksi254** [comment]
Hi @vincerubinetti @jananiravi I have done my research on making use of nightingale for MSA and I approve. Should I go ahead and build it? What do you advice?

**Comment by jananiravi** [comment]
Yes, please add a PR with your implementation. Vince and others will look it over when they get a chance. thanks!

**Comment by triksi254** [comment]
Okay, Thank you

**Comment by triksi254** [comment]
@vincerubinetti @jananiravi please allow push request

**Comment by vincerubinetti** [comment]
Closed by #50
